backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -232,81 +232,83 @@ class SchedulingService implements ISchedulingService {
         }
       }
 
-      const emailBody = `
-      <html >
+      let volunteerTimeString = "";
+      if (schedule.volunteerTime) {
+        const timePartArray = schedule.volunteerTime.split(":");
+        const AmOrPm = parseInt(timePartArray[0], 10) >= 12 ? "PM" : "AM";
+        const hour = parseInt(timePartArray[0], 10) % 12 || 12;
+        timePartArray[0] = String(hour);
+        volunteerTimeString = `${timePartArray.join(":")} ${AmOrPm}`;
+      }
```

Comment:
so this works, but it would be cleaner if we levaraged the `dayjs` library we're using on the backend 
```suggestion
    import customParseFormat from 'dayjs/plugin/customParseFormat';
    ....
     dayjs.extend(customParseFormat);
      const volunteerTimeString = dayjs(schedule.volunteerTime, "HH:mm").format("h:mm A") ?? "";
```